hibernate-core/src/test/java/org/hibernate/orm/test/query/criteria/CriteriaWindowFunctionTest.java

```diff
@@ -214,6 +215,7 @@ public void testRank(SessionFactoryScope scope) {
 
 	@Test
 	@SkipForDialect(dialectClass = DB2Dialect.class, majorVersion = 10, reason = "No support for percent_rank and cume_dist functions")
+	@SkipForDialect(dialectClass = AltibaseDialect.class, reason = "Altibase needs additional WITHIN clause in percent_rank and cume_dist functions")
```

Comment:
Oracle uses the `within group` syntax as well, so you should be able to use `commonFunctionFactory.hypotheticalOrderedSetAggregates()` just like the `OracleDialect` does.

Reply:
I have already inserted hypotheticalOrderedSetAggregates() in AltibaseDialect, but this test create query likes below and Altibase throws an error because percent_rank(), cume_dist() does not have within clause. Any ideas?

select
    row_number() over(partition by eob1_0.theInt order by eob1_0.id),
    percent_rank() over(partition by eob1_0.theInt order by eob1_0.id),
    cume_dist() over(partition by eob1_0.theInt order by eob1_0.id)
from
    EntityOfBasics eob1_0
order by 1;

Reply:
I just checked your documentation and it seems Altibase simply doesn't support the `over` clause for `percent_rank` and `cume_dist`. Please alter the reason to reflect this fact.

Reply:
Yeah. I'm gonna edit comments with more details.